src/lsm/manifest_log.zig

```diff
@@ -42,15 +39,6 @@ const RingBuffer = @import("../ring_buffer.zig").RingBuffer;
 const schema = @import("schema.zig");
 const TableInfo = schema.Manifest.TableInfo;
 
-comptime {
-    // We need enough manifest blocks to accommodate the upper-bound of tables.
-    // *10 is an arbitrary lower-bound of available space for amplification, which allows compaction
-    // to be spaced more efficiently.
-    const block_entries = schema.Manifest.entry_count_max;
-    const manifest_entries = block_entries * vsr.superblock.manifest_blocks_max;
-    assert(manifest_entries >= tree.table_count_max * 10);
-}
```

Comment:
I didn't bother updating this since it will be unnecessary (and incorrect) once the manifest block upper-bound is in place.